documentation/user_docs/docs/manual/Cutting_data_of_interest_from_SQW_files_and_objects.rst

```diff
@@ -297,290 +344,610 @@ the position of the plain you want to cut. These fields are ``u`` and ``v``:
-projection and changes from :math:`0` to :math:`180^o`. :math:`\phi` angle is measured
-from :math:`r_x` vector of the projection and changes from :math:`-180^o` to :math:`180^o`. 
-Spherical projection :math:`r_x` vector corresponds to ``sphere_proj`` class :math:`e_x` vector
-for orthogonal reciprocal lattice. If reciprocal lattice is non-orthogonal, the :math:`e_x`-vector
-of spherical projection defines the :math:`z-x` plane of spherical coordinate system and :math:`r_x` 
-vector is build in this plane but orthogonal to :math:`e_z` vector.
-
-.. figure:: ../images/spher_coordinates.png 
+
+   proj = sphere_proj([ez][, ex][, type][, alatt][, angdeg][, offset][, label][, title][, lab1][, lab2][, lab3][, lab4])
+
+where:
+
+- ``ez``
+
+  is :math:`e_z` the 3-vector defining the direction of the :math:`z`-axis in
+  reciprocal space of the spherical coordinate system.
+
+- ``ex``
+
+  is :math:`e_x` the 3-vector defining the direction of the :math:`x`-axis in
+  reciprocal space of the spherical coordinate system.
+
+.. note::
+
+   The :math:`e_y` direction is not explicitly defined as part of the
+   ``sphere_proj`` and is always considered to be perpendicular to the
+   :math:`e_x`-:math:`e_z` plane.
+
+.. note::
+
+   By default a ``sphere_proj`` will define its principal axes
+   :math:`e_z` and :math:`e_x` along the :math:`hkl` directions
+   :math:`[1,0,0]` (:math:`e_z`) and :math:`[0,1,0]` (:math:`e_x`)
+   respectively.
+
+- ``type``
+
+
+  Three character string denoting the the projection normalization of each
+  dimension, one character for each, e.g. ``'add'``, ``'rrr'``, ``'pdr'``.
+
+  There are 3 possible options for the first (length) component of ``type``,
+  which are the same as those for ``line_proj``:
+
+  1. ``'a'``
+
+     Inverse angstroms
+
+  2. ``'r'``
+
+     Reciprocal lattice units (r.l.u.) which normalises so that the maximum of
+     :math:`|h|`, :math:`|k|` and :math:`|l|` is unity.
+
+  3. ``'p'``
+
+     Preserve the values of ``u`` and ``v``
+
+  There are 2 possible options for the second and third (angular) components of
+  type:
+
+  1. ``'d'``
+
+     Degrees
+
+  2. ``'r'``
+
+     Radians
+
+  For example, if we wanted the **Q**-component to be in r.l.u. and
+  the angles in degrees we would have ``type = 'rdd'``.
+
+- ``alatt``
+
+  3-vector of lattice parameters.
+
+- ``angdeg``
+
+  3-vector of lattice angles in degrees.
+
+.. note::
+
+   In general, you should not need to define ``alatt`` or ``angdeg``;
+   by default they will be taken from the ``sqw`` object during a
+   ``cut``. However, there are cases where a projection object may
+   need to be reused elsewhere.
+
+- ``offset``
+
+  3-vector in :math:`(h,k,l)` or 4-vector in :math:`(h,k,l,e)` defining the
+  origin of the projection coordinate system.
+
+.. warning::
+
+   Prior to Horace 4.0.0 a similar (mis-named) property ``uoffset`` was
+   used. This will be ignored in Horace 4.0.0, so ensure that your scripts have
+   been updated.
```

Comment:
I have reconciled that so `offset` in Horace-4 and `uoffset` in Horace-3 are mapped to the same property implicityly. This warning should be safely removed